wiki/Tournaments/OWC/2018/en.md

```diff
@@ -0,0 +1,225 @@
+    - Declaring the match as Lost by Default
+    - Disqualification from the entire tournament
+    - Disqualification from the current and future official tournaments until appealed
+    - Any modification of these rules will be announced.
+
+### Tournament Registration
+
+1. Every user interested in joining their country's team signs up individually.
+   - Tournament Management will create a list of potential candidates for a country's team.
+   - Tournament Management declares one candidate to the captain of the country's team, albeit temporarily.
+   - The declared captain can form their team from the candidate list of their country.
+2. To ensure valid and serious registrations, every registered user will be checked by the Tournament Management.
+   - Every registered user will be assigned to their respective country's candidate list.
+   - To be successfully accepted on the list, you have to ensure that your global osu! performance ranking is above 5000.
+   - To be successfully accepted on the list, you have to ensure that you did not violate the [osu! community rules](/wiki/Rules) within the last 12 months.
+3. All successfully formed teams will be published after the Registration Phase.
+4. Only the 32 potentially strongest countries will participate. The potential strength of a country is determined by the online statistics of all valid candidates.
+   - If the amount of registered countries is below 32, the number might be reduced to 24, 20 or 16. The aim is always to let as many countries participate as possible!
+5. Mapset selectors must not participate as a player in this tournament.
+
+### Stage Instructions
+
+1. In the first stage (Group Stage), the teams will be divided into 8 groups of 4 teams.
+   - This may change depending on how many teams are accepted into the competition at the end.
+2. All the teams from each group will face each other.
+3. Rankings of each group are determined by sorting the results of each team's performance in the following priority:
+   - Most matches won.
+   - Have higher `{(the number of beatmaps won) - (the number of beatmaps defeated)}`.
+   - Most beatmaps won.
+   - **Winner of the match played previously between the tied teams.**
+   - In the event of a triple tie:
+   	- Have higher `∑{(total score difference) / (maximum score)}`.
+   	- Winner of the rematch.
+4. The top 2 teams of each group will move on to the Knock-Out Stages.
+   -  This may change with the actual Group Stage setup.
+5. Following stages are Double Elimination Stages. This means that the winner moves to the next stage and the losing team gets moved to the Loser bracket.
+6. Based on [this image](https://puu.sh/bUq5V/f1066103b0.png), the stages are split up into the following:
+
+| Stage | Match ID |
+| -----: | -------- |
+| Round of 16 |  A, B, C, D, E, F, G, H |
+| Quarterfinals | I, J, K, L & R, S, T, U |
+| Semifinals | M, N & V, W, X, Y, Z, AA |
+| Finals | O & AB, AC, AD, AE, P, Q |
```

Comment:
We will divide finals and grand finals so it should be like
| Finals | O & AB, AC, AD |
| Grand Finals | AE, P, Q |